src/components/Editor/CustomExtensions/LinkKbArticles/ArticlePicker.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState, useEffect, useRef } from "react";
+
+import { findBy } from "neetocist";
+import Search from "neetomolecules/Search";
+import { Select, Spinner as NeetoUISpinner } from "neetoui";
+import { useTranslation } from "react-i18next";
+
+import { useArticleFetching } from "hooks/useArticleFetching";
+
+import ArticlesList from "./ArticlesList";
+
+const ArticlePicker = ({
+  mode = "modal",
+  _editor,
+  cursorPos,
+  onClose,
+  _deletedArticlesHook,
```

Comment:
These props are unused.

```suggestion
  cursorPos,
  onClose,
```